channels/jaicp/src/main/kotlin/com/justai/jaicf/channel/jaicp/dto/JaicpReplies.kt

```diff
@@ -97,6 +101,7 @@ class HangupReply(val state: String? = null) : Reply("hangup") {
 @Serializable
 class TelephonySwitchReply(
     val phoneNumber: String? = null,
+    val method: String? = null,
```

Comment:
I think it's better to use `enum class` instead of `String` with available options: `invite` and `refer` and corresponding docstings.

It's hard to guess what to write here to make in work properly without JAICP documentation

Reply:
@Denire @nikvoloshin Please look last changes. I rewrite docs and add enum class TelephonySwitchMethod